packages/components/psammead-media-player/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,161 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Media Player: AMP Entry renders a landscape container with an amp-iframe and nested amp-img 1`] = `
+.c0 {
+  padding-top: 56.25%;
+  position: relative;
+  overflow: hidden;
+}
+
+<div
+  class="c0"
+  orientation="Landscape"
```

Comment:
lol its passing `orientation="Landscape"` to the div? Is `orientation` a html thing? Might need to rename it as to not pass it to the dom

Reply:
haha oh yeah, fixed